charts/harvester-load-balancer/templates/rbac.yaml

```diff
@@ -89,6 +92,9 @@ rules:
   - apiGroups: [ "kubevirt.io" ]
     resources: [ "virtualmachineinstances" ]
     verbs: [ "get", "watch", "list" ]
+  - apiGroups: [ "management.cattle.io" ]
+    resources: [ "projects" ]
+    verbs: [ "get", "watch", "list" ]
```

Comment:
This does not exist in the bare-metal cluster.

Reply:
Is that make sense If the user enables the rancher-manager-support setting?